includes/helpers/functions/general.php

```diff
@@ -948,12 +948,15 @@ function wpbdp_get_fee_plans( $args = array() ) {
     }
 
 	if ( ! $args['admin_view'] && $args['include_free'] ) {
-		$where .= $wpdb->prepare( ' AND p.amount = %d', 0 );
+		if ( $payments_on ) {
+			$where .= $wpdb->prepare( ' AND p.amount >= %d', 0 );
```

Comment:
It looks like adding this makes the query more complicated and slower. Also, it changes the behavior. Would this work instead?
```
if ( ! $args['admin_view'] && $args['include_free'] && ! $payments_on ) {
$where .= $wpdb->prepare( ' AND p.amount = %d', 0 );
} ...
```